README.md

```diff
@@ -116,6 +116,33 @@ For more imformation about the core tests, including what tests are available, h
 * **WooCommerce E2E Environment** - https://www.npmjs.com/package/@woocommerce/e2e-environment
 * **WooCommerce E2E Core Tests** - https://www.npmjs.com/package/@woocommerce/e2e-core-tests
 
+### Examples
+
+This section provides examples of plugins and packages that are implementing WooCommerce related end-to-end tests.
+
+**Woo Manage Fraud Orders**
+
+[Woo Manage Fraud Orders](https://github.com/prasidhda/woo-manage-fraud-orders) is making use of the following packages in their end-to-end tests:
+
+* [**WooCommerce E2E Environment**](https://www.npmjs.com/package/@woocommerce/e2e-environment)
+* [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils)
+* [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api)
+* [**WooCommerce Core E2E Tests**](https://www.npmjs.com/package/@woocommerce/e2e-core-tests)
+
+Examples of how they've integrated these packages and built out tests can be found [in the specs folder](https://github.com/prasidhda/woo-manage-fraud-orders/tree/master/tests/e2e/specs) within the project.
+
+**WooCommerce Payments**
+
+WooCommerce Payments is using both the [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils) and [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api) packages as part of the [WooCommerce Payments end-to-end test suite](https://github.com/Automattic/woocommerce-payments/tree/develop/tests/e2e). An example usage can be found in the [`Order > Refund Failure` test suite](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/specs/merchant/merchant-orders-refund-failures.spec.js).
```

Comment:
Same URL error as above comment.

Also, The formatting for listing packages should be the same throughout the section.

Reply:
Done, I adjusted the list of packages used to be in a bulleted-list and re-worked the paragraph a bit to accommodate the changes. Let me know what you think.